### PR TITLE
Updated events documentation to use new event system that uses getEventHandler

### PR DESCRIPTION
**Pull request by MAXOUXAX**
### Prerequisites for Code Changes

* [x] This pull request follows the code style of the project
* [x] I have tested this feature

### Changes Proposed

* Updated the event documentation so that it uses the getEventHandler system

### CODE REVIEW

**Review by iProdigy — CHANGES_REQUESTED** [review]
Could you replace `eventManager.getEventHandler(SimpleEventHandler.class).onEvent(...)` with just `eventManager.onEvent(...)` please (it defaults to SimpleEventHandler by default now)

**Comment by MAXOUXAX** [comment]
Should be good!

File: content/docs/events/channel-change-game-event.md [hunk 1]
```diff
@@ -24,21 +24,21 @@ Requires the specific channels to be registered with the TwitchClient Helper. Se
...
-eventManager.onEvent(ChannelChangeGameEvent).subscribe { event ->
+eventManager.onEvent(ChannelChangeGameEvent, { event ->
 	System.out.println "[${event.channel.name}] is now playing ${event.gameId}!"
 }
 ```
 {{</code>}}
 {{<code Kotlin>}}
 ```kotlin
-eventManager.onEvent(ChannelChangeGameEvent::class.java).subscribe { event ->
+eventManager.onEvent(ChannelChangeGameEvent::class.java, { event ->
```
Comment (iProdigy) [inline content/docs/events/channel-change-game-event.md:41]:
```suggestion
eventManager.onEvent(ChannelChangeGameEvent::class.java) { event ->
```

File: content/docs/events/channel-change-game-event.md [hunk 2]
```diff
@@ -24,21 +24,21 @@ Requires the specific channels to be registered with the TwitchClient Helper. Se
...
-eventManager.onEvent(ChannelChangeGameEvent.class).subscribe(event -> {
+eventManager.onEvent(ChannelChangeGameEvent.class, event -> {
 	System.out.println("[" + event.getChannel().getName() + "] is now playing " + event.getGameId() + "!");
 });
 ```
 {{</code>}}
 {{<code Groovy>}}
 ```groovy
-eventManager.onEvent(ChannelChangeGameEvent).subscribe { event ->
+eventManager.onEvent(ChannelChangeGameEvent, { event ->
```
Comment (iProdigy) [inline content/docs/events/channel-change-game-event.md:34]:
```suggestion
eventManager.onEvent(ChannelChangeGameEvent) { event ->
```

File: content/docs/events/channel-change-title-event.md [hunk 3]
```diff
@@ -24,21 +24,21 @@ Requires the specific channels to be registered with the TwitchClient Helper. Se
...
-eventManager.onEvent(ChannelChangeTitleEvent.class).subscribe(event -> {
+eventManager.onEvent(ChannelChangeTitleEvent.class, event -> {
 	System.out.println("[" + event.getChannel().getName() + "] changed his stream title to " + event.getTitle() + "!");
 });
 ```
 {{</code>}}
 {{<code Groovy>}}
 ```groovy
-eventManager.onEvent(ChannelChangeTitleEvent).subscribe { event ->
+eventManager.onEvent(ChannelChangeTitleEvent, { event ->
```
Comment (iProdigy) [inline content/docs/events/channel-change-title-event.md:34]:
```suggestion
eventManager.onEvent(ChannelChangeTitleEvent) { event ->
```

File: content/docs/events/channel-change-title-event.md [hunk 4]
```diff
@@ -24,21 +24,21 @@ Requires the specific channels to be registered with the TwitchClient Helper. Se
...
 	System.out.println "[${event.channel.name}] changed his stream title to ${event.title}!"
 }
 ```
 {{</code>}}
 {{<code Kotlin>}}
 ```kotlin
-eventManager.onEvent(ChannelChangeTitleEvent::class.java).subscribe { event ->
+eventManager.onEvent(ChannelChangeTitleEvent::class.java, { event ->
```
Comment (iProdigy) [inline content/docs/events/channel-change-title-event.md:41]:
```suggestion
eventManager.onEvent(ChannelChangeTitleEvent::class.java) { event ->
```

File: content/docs/events/channel-go-live-event.md [hunk 5]
```diff
@@ -24,21 +24,21 @@ Requires the specific channels to be registered with the TwitchClient Helper. Se
...
-eventManager.onEvent(ChannelGoLiveEvent.class).subscribe(event -> {
+eventManager.onEvent(ChannelGoLiveEvent.class, event -> {
 	System.out.println("[" + event.getChannel().getName() + "] went live with title " + event.getTitle() + " on game " + event.getGameId() + "!");
 });
 ```
 {{</code>}}
 {{<code Groovy>}}
 ```groovy
-eventManager.onEvent(ChannelGoLiveEvent).subscribe { event ->
+eventManager.onEvent(ChannelGoLiveEvent, { event ->
```
Comment (iProdigy) [inline content/docs/events/channel-go-live-event.md:34]:
```suggestion
eventManager.onEvent(ChannelGoLiveEvent) { event ->
```

File: content/docs/events/channel-go-live-event.md [hunk 6]
```diff
@@ -24,21 +24,21 @@ Requires the specific channels to be registered with the TwitchClient Helper. Se
...
 	System.out.println "[${event.channel.name}] went live with title ${event.title} on game ${event.gameId}!"
 }
 ```
 {{</code>}}
 {{<code Kotlin>}}
 ```kotlin
-eventManager.onEvent(ChannelGoLiveEvent::class.java).subscribe { event ->
+eventManager.onEvent(ChannelGoLiveEvent::class.java, { event ->
```
Comment (iProdigy) [inline content/docs/events/channel-go-live-event.md:41]:
```suggestion
eventManager.onEvent(ChannelGoLiveEvent::class.java) { event ->
```

File: content/docs/events/channel-go-offline-event.md [hunk 7]
```diff
@@ -24,21 +24,21 @@ Requires the specific channels to be registered with the TwitchClient Helper. Se
...
-eventManager.onEvent(ChannelGoOfflineEvent.class).subscribe(event -> {
+eventManager.onEvent(ChannelGoOfflineEvent.class, event -> {
 	System.out.println("[" + event.getChannel().getName() + "] just went offline!");
 });
 ```
 {{</code>}}
 {{<code Groovy>}}
 ```groovy
-eventManager.onEvent(ChannelGoOfflineEvent).subscribe { event ->
+eventManager.onEvent(ChannelGoOfflineEvent, { event ->
```
Comment (iProdigy) [inline content/docs/events/channel-go-offline-event.md:34]:
```suggestion
eventManager.onEvent(ChannelGoOfflineEvent) { event ->
```

File: content/docs/events/channel-go-offline-event.md [hunk 8]
```diff
@@ -24,21 +24,21 @@ Requires the specific channels to be registered with the TwitchClient Helper. Se
...
 	System.out.println "[${event.channel.name}] just went offline!"
 }
 ```
 {{</code>}}
 {{<code Kotlin>}}
 ```kotlin
-eventManager.onEvent(ChannelGoOfflineEvent::class.java).subscribe { event -> 
+eventManager.onEvent(ChannelGoOfflineEvent::class.java, { event ->
```
Comment (iProdigy) [inline content/docs/events/channel-go-offline-event.md:41]:
```suggestion
eventManager.onEvent(ChannelGoOfflineEvent::class.java) { event -> 
```

File: content/docs/events/channel-message-event.md [hunk 9]
```diff
@@ -24,21 +24,21 @@ Requires the TwitchChat Module to be enabled and active for the source channel.
...
-eventManager.onEvent(ChannelMessageEvent.class).subscribe(event -> {
+eventManager.onEvent(ChannelMessageEvent.class, event -> {
 	System.out.println("[" + event.getChannel().getName() + "] " + event.getUser().getName() + ": " + event.getMessage());
 });
 ```
 {{</code>}}
 {{<code Groovy>}}
 ```groovy
-eventManager.onEvent(ChannelMessageEvent).subscribe { event ->
+eventManager.onEvent(ChannelMessageEvent, { event ->
```
Comment (iProdigy) [inline content/docs/events/channel-message-event.md:34]:
```suggestion
eventManager.onEvent(ChannelMessageEvent) { event ->
```

File: content/docs/events/channel-message-event.md [hunk 10]
```diff
@@ -24,21 +24,21 @@ Requires the TwitchChat Module to be enabled and active for the source channel.
...
 	System.out.println "[${event.channel.name}] ${event.user.name}: ${event.message}"
 }
 ```
 {{</code>}}
 {{<code Kotlin>}}
 ```kotlin
-eventManager.onEvent(ChannelMessageEvent::class.java).subscribe { event ->
+eventManager.onEvent(ChannelMessageEvent::class.java, { event ->
```
Comment (iProdigy) [inline content/docs/events/channel-message-event.md:41]:
```suggestion
eventManager.onEvent(ChannelMessageEvent::class.java) { event ->
```

File: content/docs/events/irc-message-event.md [hunk 11]
```diff
@@ -24,21 +24,21 @@ Requires the TwitchChat Module to be enabled and active for the source channel.
...
-eventManager.onEvent(IRCMessageEvent.class).subscribe(event -> {
+eventManager.onEvent(IRCMessageEvent.class, event -> {
 	System.out.println(event.getRawMessage());
 });
 ```
 {{</code>}}
 {{<code Groovy>}}
 ```groovy
-eventManager.onEvent(IRCMessageEvent).subscribe { event ->
+eventManager.onEvent(IRCMessageEvent, { event ->
```
Comment (iProdigy) [inline content/docs/events/irc-message-event.md:34]:
```suggestion
eventManager.onEvent(IRCMessageEvent) { event ->
```

File: content/docs/events/irc-message-event.md [hunk 12]
```diff
@@ -24,21 +24,21 @@ Requires the TwitchChat Module to be enabled and active for the source channel.
...
 	System.out.println event.rawMessage
 }
 ```
 {{</code>}}
 {{<code Kotlin>}}
 ```kotlin
-eventManager.onEvent(IRCMessageEvent::class.java).subscribe { event ->
+eventManager.onEvent(IRCMessageEvent::class.java, { event ->
```
Comment (iProdigy) [inline content/docs/events/irc-message-event.md:41]:
```suggestion
eventManager.onEvent(IRCMessageEvent::class.java) { event ->
```

File: content/docs/events/private-message-event.md [hunk 13]
```diff
@@ -25,21 +25,21 @@ Otherwise you will need to enable [PubSub](../pubsub) and subscribe to the whisp
...
-eventManager.onEvent(PrivateMessageEvent.class).subscribe(event -> {
+eventManager.onEvent(PrivateMessageEvent.class, event -> {
 	System.out.println("[Whisper] " + event.getUser().getName() + ": " + event.getMessage());
 });
 ```
 {{</code>}}
 {{<code Groovy>}}
 ```groovy
-eventManager.onEvent(PrivateMessageEvent).subscribe { event ->
+eventManager.onEvent(PrivateMessageEvent, { event ->
```
Comment (iProdigy) [inline content/docs/events/private-message-event.md:35]:
```suggestion
eventManager.onEvent(PrivateMessageEvent) { event ->
```

File: content/docs/events/private-message-event.md [hunk 14]
```diff
@@ -25,21 +25,21 @@ Otherwise you will need to enable [PubSub](../pubsub) and subscribe to the whisp
...
 	System.out.println "[Whisper] ${event.user.name}: ${event.message}"
 }
 ```
 {{</code>}}
 {{<code Kotlin>}}
 ```kotlin
-eventManager.onEvent(PrivateMessageEvent::class.java).subscribe { event ->
+eventManager.onEvent(PrivateMessageEvent::class.java, { event ->
```
Comment (iProdigy) [inline content/docs/events/private-message-event.md:42]:
```suggestion
eventManager.onEvent(PrivateMessageEvent::class.java) { event ->
```

**Review by iProdigy — COMMENTED** [review]
utilize SAM interfaces for kotlin and groovy

**Review by iProdigy — APPROVED** [review]

**Comment by iProdigy** [comment]
Thank you!

**Pull request merged**